Kickstarter-iOS/Views/Controllers/PledgeTableViewController.swift

```diff
@@ -51,10 +52,30 @@ class PledgeTableViewController: UITableViewController {
 
     self.viewModel.outputs.reloadWithData
       .observeForUI()
-      .observeValues { [weak self] (amount, currency, delivery, isLoggedIn) in
-        self?.dataSource.load(amount: amount, currency: currency, delivery: delivery, isLoggedIn: isLoggedIn)
+      .observeValues { [weak self] data in
+        self?.dataSource.load(data: data)
 
         self?.tableView.reloadData()
+
+        self?.viewModel.inputs.reloadData()
```

Comment:
So the `output` of the VM is feeding the `input` of the VM again? Is this necessary? Can't `reloadWithData` trigger `selectedShippingRulesData`?

Reply:
Or is this because it needs to happen after the table view is reloaded?

Reply:
I think I finally see it - it was because of https://github.com/kickstarter/ios-oss/pull/682/files#r287428427 wasn't it?